lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -226,7 +226,13 @@ class LighthouseReportViewer {
     return new Promise((resolve, reject) => {
       const reader = new FileReader();
       reader.onload = function(e) {
-        resolve(e.target && e.target.result);
```

Comment:
technically it's somewhat more correct. They had a custom `FileReaderLoadEvent` passed to `onload` before, which always has the target set to the `FileReader`, but that isn't a real standardized event. They're moving to automatically generate the types from Edge's Web IDL files, so I think that's why that custom event was eliminated.

However, the event *should* be generic, parameterized on the element itself so that the type is known and it isn't just an `EventTarget` but the actual element, but they haven't done that. Sort of for performance reasons, but probably also partly because people aren't asking hard enough: https://github.com/Microsoft/TSJS-lib-generator/pull/207